.travis.yml

```diff
@@ -9,6 +9,7 @@ script:
   - RAILS_ENV=test bundle exec rake db:drop
   - RAILS_ENV=test bundle exec rake db:create
   - RAILS_ENV=test bundle exec rake db:migrate
+  - bundle exec brakeman -zq --ensure-latest
```

Comment:
the `--ensure-latest` will make the build fail when there is a new version of brakeman

IMO this is a fair price to pay for keeping brakeman up to date at all times, but let me know if you prefer to leave that out